ERROR - 2022-12-24 04:35:05 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'lumiere' in C:\wamp64\www\AtoZ\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\wamp64\www\AtoZ\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'lumiere', 3306, '', 0)
#1 C:\wamp64\www\AtoZ\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp64\www\AtoZ\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp64\www\AtoZ\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\wamp64\www\AtoZ\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\AtoZ\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\wamp64\www\AtoZ\app\Controllers\Package.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\wamp64\www\AtoZ\system\CodeIgniter.php(925): App\Controllers\Package->packages()
#8 C:\wamp64\www\AtoZ\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Package))
#9 C:\wamp64\www\AtoZ\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\wamp64\www\AtoZ\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\wamp64\www\AtoZ\system\Commands\Server\rewrite.php(47): require_once('C:\\wamp64\\www\\A...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'lumiere' in C:\wamp64\www\AtoZ\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\wamp64\www\AtoZ\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp64\www\AtoZ\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp64\www\AtoZ\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\wamp64\www\AtoZ\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp64\www\AtoZ\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\wamp64\www\AtoZ\app\Controllers\Package.php(12): CodeIgniter\BaseModel->findAll()
#6 C:\wamp64\www\AtoZ\system\CodeIgniter.php(925): App\Controllers\Package->packages()
#7 C:\wamp64\www\AtoZ\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Package))
#8 C:\wamp64\www\AtoZ\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\AtoZ\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\wamp64\www\AtoZ\system\Commands\Server\rewrite.php(47): require_once('C:\\wamp64\\www\\A...')
#11 {main}
CRITICAL - 2022-12-24 04:35:05 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'lumiere'
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `packages`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Package.php(12): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Package->packages()
 7 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Package))
 8 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-24 19:25:04 --> Undefined variable $url
in APPPATH\Controllers\Auth.php on line 84.
 1 APPPATH\Controllers\Auth.php(84): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $url', 'APPPATH\\Controllers\\Auth.php', 84)
 2 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-24 19:49:20 --> Undefined variable $url
in APPPATH\Controllers\Auth.php on line 84.
 1 APPPATH\Controllers\Auth.php(84): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $url', 'APPPATH\\Controllers\\Auth.php', 84)
 2 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-24 19:49:53 --> Undefined variable $url
in APPPATH\Controllers\Auth.php on line 84.
 1 APPPATH\Controllers\Auth.php(84): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $url', 'APPPATH\\Controllers\\Auth.php', 84)
 2 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-24 19:50:12 --> Undefined variable $url
in APPPATH\Controllers\Auth.php on line 84.
 1 APPPATH\Controllers\Auth.php(84): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $url', 'APPPATH\\Controllers\\Auth.php', 84)
 2 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-24 20:39:55 --> Undefined variable $url
in APPPATH\Controllers\Auth.php on line 84.
 1 APPPATH\Controllers\Auth.php(84): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $url', 'APPPATH\\Controllers\\Auth.php', 84)
 2 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-24 20:53:14 --> Undefined property: App\Controllers\Auth::$Request
in APPPATH\Controllers\Auth.php on line 54.
 1 APPPATH\Controllers\Auth.php(54): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Auth::$Request', 'APPPATH\\Controllers\\Auth.php', 54)
 2 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Auth->login()
 3 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
